integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -31,7 +31,7 @@ object HiveExample {
 
   def main(args: Array[String]) {
```

Comment:
Please optimize this example and RunHiveExampleTest like:org.apache.carbondata.examplesCI.RunExamples.
 We should add the example to CI, and add some assert for exception and result. We should avoid that some developer/PR change code and lead to the hiveExample error,  the developer need fix it before their PR merged

Reply:
I updated RunHiveExampleTest and RunExamples as same as RunExamples.But this doesn't work in CI.I guess that the problem is caused by CI.